code/datums/status_effects/debuffs/blindness.dm

```diff
@@ -37,7 +42,8 @@
 		var/mob/living/carbon/human/human_owner = owner
 		if (human_owner.get_eye_scars())
 			return TRUE
-	return !HAS_TRAIT(owner, TRAIT_NEARSIGHTED_CORRECTED)
+	var/good_eyesight = HAS_TRAIT(owner, TRAIT_NEARSIGHTED_CORRECTED) || HAS_TRAIT(owner, TRAIT_SIGHT_BYPASS)
+	return !good_eyesight
```

Comment:
Work smarter, not harder?
```suggestion
	var/good_eyesight = !HAS_TRAIT(owner, TRAIT_NEARSIGHTED_CORRECTED) && !HAS_TRAIT(owner, TRAIT_SIGHT_BYPASS)
```

Reply:
If you're concerned about readability of this I'd just split it into ifs
```dm
	if(HAS_TRAIT(owner, TRAIT_NEARSIGHTED_CORRECTED))
		return TRUE
	if(HAS_TRAIT(owner, TRAIT_SIGHT_BYPASS))
		return TRUE
	return FALSE
```

Reply:
why dont we have a HAS_TRAIT_IN_LIST helper